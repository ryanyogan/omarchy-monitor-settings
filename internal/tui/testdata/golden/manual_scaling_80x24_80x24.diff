Visual regression detected!

=== EXPECTED ===
  [90m╭────────────────────────────────────────────────────────────────────────╮[0m    
  [90m│[0m                                                                        [90m│[0m    
  [90m│[0m                            [1;34mDisplay Settings[0m                            [90m│[0m    
  [90m│[0m                                                                        [90m│[0m    
  [90m╰────────────────────────────────────────────────────────────────────────╯[0m    
                                                                                
  [35m╭────────────────────────────────────────────────────────────────────────╮[0m    
  [35m│[0m                                                                        [35m│[0m    
  [35m│[0m                                                                        [35m│[0m    
  [35m│[0m  [1;35m🔧 Manual Scaling Controls[0m                                            [35m│[0m    
  [35m│[0m                                                                        [35m│[0m    
  [35m│[0m  [33m╭──────────────────────────────────────────────────────────╮[0m          [35m│[0m    
  [35m│[0m  [33m│[0m[40m                                                          [0m[33m│[0m          [35m│[0m    
  [35m│[0m  [33m│[0m[40m  [0m[40mHDMI-A-1 Dell U2414H - 1920x1080@60Hz (Current: 1.00x)[0m[40m  [0m[33m│[0m          [35m│[0m    
  [35m│[0m  [33m│[0m[40m                                                          [0m[33m│[0m          [35m│[0m    
  [35m│[0m  [33m╰──────────────────────────────────────────────────────────╯[0m          [35m│[0m    
  [35m│[0m                                                                        [35m│[0m    
  [35m│[0m  [1;34m⚙️ Scaling Controls[0m                                                   [35m│[0m    
  [35m│[0m                                                                        [35m│[0m    
  [35m│[0m  [40m [0m[1;34;40m1. Monitor Scale (Compositor-level)[0m[40m [0m                                 [35m│[0m    
  [35m│[0m  [40m [0m[32;40m   Current: 1.000x (Valid: 1.0x, 1.25x, 1.33x, 1.5x, 1.67x, 1.75x,[m   [35m│[0m    
  [35m│[0m  [32;40m2.0x, 2.25x, 2.5x, 3.0x)[0m[40m [0m                                             [35m│[0m    
  [35m│[0m  [40m [0m[37;40m   Scales everything immediately. Works with all apps.[0m[40m [0m              [35m│[0m    
  [35m│[0m                                                                        [35m│[0m    
  [35m│[0m  [1;36m2. GTK Scale (Application-level)[0m                                      [35m│[0m    
  [35m│[0m  [32m   Current: 1x (Range: 1x - 3x, Integer only)[0m                         [35m│[0m    
  [35m│[0m  [37m   Scales GTK apps (most Linux apps). Requires logout.[0m                [35m│[0m    
  [35m│[0m                                                                        [35m│[0m    
  [35m│[0m  [1;33m3. Font DPI (Text rendering)[0m                                          [35m│[0m    
  [35m│[0m  [32m   Current: 96 (Range: 72 - 288, Step: 12)[0m                            [35m│[0m    
  [35m│[0m  [37m   Fine-grained text scaling. Works with most applications.[0m           [35m│[0m    
  [35m│[0m                                                                        [35m│[0m    
  [35m│[0m  [1;32m📊 Preview Results[0m                                                    [35m│[0m    
  [35m│[0m    Effective Resolution: 1920x1080                                     [35m│[0m    
  [35m│[0m    Screen Real Estate: 100%                                            [35m│[0m    
  [35m│[0m    Font DPI Multiplier: 1.0x                                           [35m│[0m    
  [35m│[0m                                                                        [35m│[0m    
  [35m│[0m  [32m↑↓[0m[37m select control[0m  [36m←→[0m[37m adjust value[0m  [33m⏎[0m[37m apply all[0m  [35mesc[0m[37m back[0m             [35m│[0m    
  [35m│[0m                                                                        [35m│[0m    
  [35m│[0m                                                                        [35m│[0m    
  [35m╰────────────────────────────────────────────────────────────────────────╯[0m    
                                                                                
  [90m╭────────────────────────────────────────────────────────────────────────╮[0m    
  [90m│[0m                                                                        [90m│[0m    
  [90m│[0m       [1;32m↑↓[0m  [37mnavigate[0m    [1;34m⏎[0m  [37mselect[0m    [1;33mh[0m  [37mhelp[0m    [1;35mesc[0m  [37mback[0m    [1;31mq[0m  [37mquit[0m     [90m│[0m    
  [90m│[0m                                                                        [90m│[0m    
  [90m╰────────────────────────────────────────────────────────────────────────╯[0m    

=== ACTUAL ===
  ╭────────────────────────────────────────────────────────────────────────╮    
  │                                                                        │    
  │                            Display Settings                            │    
  │                                                                        │    
  ╰────────────────────────────────────────────────────────────────────────╯    
                                                                                
  ╭────────────────────────────────────────────────────────────────────────╮    
  │                                                                        │    
  │                                                                        │    
  │  🔧 Manual Scaling Controls                                            │    
  │                                                                        │    
  │  ╭──────────────────────────────────────────────────────────╮          │    
  │  │                                                          │          │    
  │  │  HDMI-A-1 Dell U2414H - 1920x1080@60Hz (Current: 1.00x)  │          │    
  │  │                                                          │          │    
  │  ╰──────────────────────────────────────────────────────────╯          │    
  │                                                                        │    
  │  ⚙️ Scaling Controls                                                   │    
  │                                                                        │    
  │   1. Monitor Scale (Compositor-level)                                  │    
  │      Current: 1.000x (Valid: 1.0x, 1.25x, 1.33x, 1.5x, 1.67x, 1.75x,   │    
  │  2.0x, 2.25x, 2.5x, 3.0x)                                              │    
  │      Scales everything immediately. Works with all apps.               │    
  │                                                                        │    
  │  2. GTK Scale (Application-level)                                      │    
  │     Current: 1x (Range: 1x - 3x, Integer only)                         │    
  │     Scales GTK apps (most Linux apps). Requires logout.                │    
  │                                                                        │    
  │  3. Font DPI (Text rendering)                                          │    
  │     Current: 96 (Range: 72 - 288, Step: 12)                            │    
  │     Fine-grained text scaling. Works with most applications.           │    
  │                                                                        │    
  │  📊 Preview Results                                                    │    
  │    Effective Resolution: 1920x1080                                     │    
  │    Screen Real Estate: 100%                                            │    
  │    Font DPI Multiplier: 1.0x                                           │    
  │                                                                        │    
  │  ↑↓ select control  ←→ adjust value  ⏎ apply all  esc back             │    
  │                                                                        │    
  │                                                                        │    
  ╰────────────────────────────────────────────────────────────────────────╯    
                                                                                
  ╭────────────────────────────────────────────────────────────────────────╮    
  │                                                                        │    
  │       ↑↓  navigate    ⏎  select    h  help    esc  back    q  quit     │    
  │                                                                        │    
  ╰────────────────────────────────────────────────────────────────────────╯    

=== DIFF INFO ===
Expected length: 6376
Actual length: 5126
Expected hash: 590fcd2f810643b7509b5e10a84cca64d904a4c8b28dc22f1eace94bdf84b456
Actual hash: 3935c522f7dee7602a85dfbf92349e1a671299e4378432fd389a7eb157944e6f
