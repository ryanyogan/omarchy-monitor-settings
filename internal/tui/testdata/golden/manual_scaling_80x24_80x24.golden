# Visual Golden File
# Name: manual_scaling_80x24
# Dimensions: 80x24
# Hash: 3935c522f7dee7602a85dfbf92349e1a671299e4378432fd389a7eb157944e6f

  ╭────────────────────────────────────────────────────────────────────────╮    
  │                                                                        │    
  │                            Display Settings                            │    
  │                                                                        │    
  ╰────────────────────────────────────────────────────────────────────────╯    
                                                                                
  ╭────────────────────────────────────────────────────────────────────────╮    
  │                                                                        │    
  │                                                                        │    
  │  🔧 Manual Scaling Controls                                            │    
  │                                                                        │    
  │  ╭──────────────────────────────────────────────────────────╮          │    
  │  │                                                          │          │    
  │  │  HDMI-A-1 Dell U2414H - 1920x1080@60Hz (Current: 1.00x)  │          │    
  │  │                                                          │          │    
  │  ╰──────────────────────────────────────────────────────────╯          │    
  │                                                                        │    
  │  ⚙️ Scaling Controls                                                   │    
  │                                                                        │    
  │   1. Monitor Scale (Compositor-level)                                  │    
  │      Current: 1.000x (Valid: 1.0x, 1.25x, 1.33x, 1.5x, 1.67x, 1.75x,   │    
  │  2.0x, 2.25x, 2.5x, 3.0x)                                              │    
  │      Scales everything immediately. Works with all apps.               │    
  │                                                                        │    
  │  2. GTK Scale (Application-level)                                      │    
  │     Current: 1x (Range: 1x - 3x, Integer only)                         │    
  │     Scales GTK apps (most Linux apps). Requires logout.                │    
  │                                                                        │    
  │  3. Font DPI (Text rendering)                                          │    
  │     Current: 96 (Range: 72 - 288, Step: 12)                            │    
  │     Fine-grained text scaling. Works with most applications.           │    
  │                                                                        │    
  │  📊 Preview Results                                                    │    
  │    Effective Resolution: 1920x1080                                     │    
  │    Screen Real Estate: 100%                                            │    
  │    Font DPI Multiplier: 1.0x                                           │    
  │                                                                        │    
  │  ↑↓ select control  ←→ adjust value  ⏎ apply all  esc back             │    
  │                                                                        │    
  │                                                                        │    
  ╰────────────────────────────────────────────────────────────────────────╯    
                                                                                
  ╭────────────────────────────────────────────────────────────────────────╮    
  │                                                                        │    
  │       ↑↓  navigate    ⏎  select    h  help    esc  back    q  quit     │    
  │                                                                        │    
  ╰────────────────────────────────────────────────────────────────────────╯    